packages/modules-sdk/src/remote-query.ts

```diff
@@ -48,10 +47,17 @@ export class RemoteQuery {
       }
 
       this.modulesMap.set(serviceName, mod)
-      servicesConfig!.push(mod.__joinerConfig)
+      const serviceIndex = servicesConfig!.findIndex(sc => sc.serviceName === mod.__joinerConfig.serviceName)
```

Comment:
todo: I ll have a review tomorrow morning unless @carlos-r-l-rodrigues review it before me 😅 But this section should be removed. It is trying to work around a bug which is that here the reference of serviceConfig is mutated and therefore impact the number of elements in it later on. We fixed it in our pr and in reality we should use a copy of serviceConfig here before pushing